docs/src/pages/[platform]/components/pagination/examples/PaginationAccessibilityExample.tsx

```diff
@@ -0,0 +1,14 @@
+import { usePagination, Pagination } from '@aws-amplify/ui-react';
+
+export const PaginationAccessibilityExample = () => {
+  const paginationProps = usePagination({ totalPages: 5 });
+  return (
+    <Pagination
+      pageLabel="Jump to page"
+      currentPageLabel="You are on page"
+      leftArrowLabel="Back to previous page"
+      rightArrowLabel="Forward to next page"
```

Comment:
Can we change these to `nextLabel` and `previousLabel` so they make sense for RTL?

Reply:
Thanks for the suggestion! I am kinda on the fence when naming this two. I was actually thinking either `prev` or `next` might be confusing because they can be sibling buttons  of current page semantically versus `<` and `>` buttons. But I am good with `next` and `previous`